### PR TITLE
test: Product Relation Mapper

### PR DESCRIPTION
**Pull request by wfajczyk**

### CODE REVIEW

File: tests/Infrastructure/Mapper/AbstractProductRelationAttributeCase.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+use Ergonode\Channel\Domain\Entity\Export;
+use Ergonode\Core\Domain\ValueObject\Language;
+use Ergonode\ExporterShopware6\Domain\Entity\Shopware6Channel;
+use Ergonode\ExporterShopware6\Infrastructure\Model\AbstractProductCrossSelling;
+use Ergonode\Product\Domain\Entity\AbstractProduct;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class AbstractProductRelationAttributeCase extends TestCase
```
Comment (piotrkreft) [inline tests/Infrastructure/Mapper/AbstractProductRelationAttributeCase.php:19]:
```suggestion
abstract class AbstractProductRelationMapperAttributeCase extends TestCase
```

Comment (piotrkreft) [reply tests/Infrastructure/Mapper/AbstractProductRelationAttributeCase.php:19]: Actually, this should fail(via warning) as you have no tests in your class

**Review by wiewiurdp — APPROVED** [review]

File: tests/Infrastructure/Mapper/AbstractProductRelationAttributeCase.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            ->willReturn(new Language('en_GB'));
+
+        $this->export = $this->createMock(Export::class);
+
+        $this->product = $this->createMock(AbstractProduct::class);
+    }
+
+    protected function getProductCrossSellingClass(): AbstractProductCrossSelling
+    {
+        return new class(
```
Comment (piotrkreft) [inline tests/Infrastructure/Mapper/AbstractProductRelationAttributeCase.php:49]: Weirdo formatting

File: tests/Infrastructure/Mapper/ProductRelationAttribute/ChildrenMapperTest.php [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    {
+        parent::setUp();
+
+        $this->calculator = $this->createMock(TranslationInheritanceCalculator::class);
+        $this->shopware6ProductRepository = $this->createMock(ProductRepositoryInterface::class);
+    }
+
+    public function testNoAttribute(): void
+    {
+        $this->product->expects(self::once())->method('hasAttribute')->willReturn(false);
```
Comment (piotrkreft) [inline tests/Infrastructure/Mapper/ProductRelationAttribute/ChildrenMapperTest.php:45]: It's a fake test. It should test method output, not the fact a method is called once

**Review by piotrkreft — CHANGES_REQUESTED** [review]

File: tests/Infrastructure/Mapper/AbstractProductRelationAttributeCase.php (hunk 1 above)
Comment (piotrkreft) [reply tests/Infrastructure/Mapper/AbstractProductRelationAttributeCase.php:19]: Ahhh it does not have Test suffix so that's the reason probably :)

**Review by piotrkreft — APPROVED** [review]

**Review by wiewiurdp — APPROVED** [review]

**Pull request merged**